cli/command/trust/inspect.go

```diff
@@ -11,24 +12,56 @@ import (
 	"github.com/theupdateframework/notary/tuf/data"
 )
 
+type inspectOptions struct {
+	remotes []string
+	/* FIXME(n4ss): this is consistent with `docker service inspect` but we should provide
```

Comment:
go comments are generally done with `//` even if they are multi-line. `/* */` is only used for package comments.